packages/facility-server/app/routes/apiv1/programRegistry.js

```diff
@@ -186,6 +186,7 @@ programRegistry.get(
             WHERE program_registry_id = :programRegistryId
           ) n
           WHERE n.row_num = 1
+          AND is_most_recent IS TRUE
```

Comment:
This query is to get most recent registrations for patients, it should consider the `is_most_recent` flag instead of just getting the latest registration by time